packages/vkui/src/components/Button/Button.test.tsx

```diff
@@ -39,4 +39,16 @@ describe('Button', () => {
     fireEvent.click(button());
     expect(handleClick).toHaveBeenCalledTimes(0);
   });
+
+  it('should handle zero in props', () => {
+    render(
+      <ButtonTest before={0} after={0}>
+        0
+      </ButtonTest>,
+    );
+    const elButton = button();
+    expect(elButton).toContainElement(screen.getByTestId('before'));
+    expect(elButton).toContainElement(screen.getByTestId('children'));
+    expect(elButton).toContainElement(screen.getByTestId('after'));
```

Comment:
Если getByTestId не найдет элемент, то тест сразу упадет

```suggestion
    screen.getByTestId('before');
    screen.getByTestId('children');
    screen.getByTestId('after');
```